go-controller/pkg/ovn/master_test.go

```diff
@@ -111,182 +109,6 @@ func (n tNode) logicalSwitch(loadBalancerGroupUUID string) *nbdb.LogicalSwitch {
 	}
 }
 
-/*
```

Comment:
I want to say these were still here from Tim's conversion to Shared GW... I.e maybe they are still relevant and just need to be fixed... I'm not 100% sure

Reply:
Yeah, some might still be relevant, but they are so out of date at this point that it would be pretty hard. That said, I could drop that commit and keep them around?

Reply:
Dropped that commit.